arrayobject/index.ts

```diff
@@ -12,6 +12,40 @@ export const getKeyValues = (data: Array<Record<string, unknown>>, key: string):
     return res;
 }
 
+export const charsIncluded = (data: string, chars: string[]): number => {
+    let counter: number = 0;
+    for (let i = 0; i < chars.length; i++) {
+        if(data.includes(chars[i])) {
+            counter++;
+        }
+    }
+    return counter;
+}
+
+export const search = (data: Array<Record<string, object>>, value: string): Array<Record<string, object>> => {
```

Comment:
change this to countCharsInDataEx

Reply:
countCharsInDataEx? I don't get this please

Reply:
The question will be search what?.. power function names should be explicit. You could tell that we can have different search algorithms and this is not an abstract class Search that all algs should extend. Got it?

Reply:
okay.. Well, it searches through all objects. So which name do you suggest because it can't be countCharsInDataEx as it doesn't count chars

Reply:
SortObjectValueBychars

Reply:
This algorithm can be better though, use loop if string.includes is false.

Reply:
FYI: string.includes is case-insensitive

Reply:
> SortObjectValuesBychars

dope